FlattenTransparency/src/main/kotlin/com/datalogics/pdfl/samples/FlattenTransparency.kt

```diff
@@ -0,0 +1,102 @@
+package com.datalogics.pdfl.samples
+
+import com.datalogics.PDFL.*
+import java.util.*
+
+/*
+ * This sample shows how to flatten transparencies in a PDF document.
+ *
+ * PDF files can have objects that are partially or fully transparent, and thus
+ * can blend in various ways with objects behind them. Transparent graphics or images
+ * can be stacked in a PDF file, with each one contributing to the final result that
+ * appears on the page. The process to flatten a set of transparencies merges them
+ * into a single image on the page.
+ *
+ * Copyright (c) 2007-2024, Datalogics, Inc. All rights reserved.
```

Comment:
Sould this copyright say 2007-2024 or should it just say 2024?

Reply:
The Java sample has 2007 - 2023 as the copyright, so I just modified that for 2024. I'm alright with either, but 2024 might be the way to go.

Reply:
The copyright is supposed to cover the time the file was created until the present day, so this should just be "2024" (and get bumped to "2024-2025" next year).

@datalogics-robb As a side issue, we need to bump the copyright dates in our other samples.

Reply:
I updated the copyright date on the Kotlin samples.